README.md

```diff
@@ -452,6 +454,14 @@ sudo ln -s /path/to/cuda /usr/local/cuda-X.X
 Please let us know on [the issue tracker](https://github.com/google/jax/issues)
 if you run into any errors or problems with the prebuilt wheels.
 
+### pip installation: Google Cloud TPU
+JAX also provides pre-built wheels for Google Cloud TPU. To install JAX along with appropriate
```

Comment:
Maybe make "Google Cloud TPU" a link to https://cloud.google.com/tpu/docs/users-guide-tpu-vm? Or not.

Reply:
Done